.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@4.7.0
+
+jobs:
+  notify_slack:
+    docker:
+      - image: cimg/node:17.2.0
+    steps:
+      - run:
+          command: |
+            curl -X POST -H 'Content-type: application/json' --data '{"text":"New PHPCS Config Release","blocks":[{"type":"header","text":{"type":"plain_text","text":"New  PHPCS Config Release"}},{"type":"section","fields":[{"type":"mrkdwn","text":"*Version:* <https://github.com/'${CIRCLE_PROJECT_USERNAME}'/'${CIRCLE_PROJECT_REPONAME}'/releases/tag/'${CIRCLE_TAG}'|'${CIRCLE_TAG}'>"},{"type":"mrkdwn","text":"<https://github.com/'${CIRCLE_PROJECT_USERNAME}'/'${CIRCLE_PROJECT_REPONAME}'/issues|Issues> *|* <https://github.com/'${CIRCLE_PROJECT_USERNAME}'/'${CIRCLE_PROJECT_REPONAME}'/pulls|Pull Requests>"}]}]}' ${SLACK_WEBHOOK}
```

Comment:
Sorry to be a pest, but there's now a double space in the message text 🙈 Might look a bit weird in Slack.

Reply:
This what I get for rushing.